src/web/components/Contributors/Contributors.css

```diff
@@ -0,0 +1,72 @@
+.ftk-contributors-title {
+    text-align: left;
+    margin-left: 36px;
+    margin-top: 32px;
+    margin-bottom: 12px;
+}
+
+.ftk-background-card {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: var(--colorNeutralBackground1);
+    background-image: url(finopstoolkitbg.png);
+    background-repeat: repeat;
+    background-size: auto;
+    background-position: top left;
+    border-radius: 12px;
+    padding: 36px;
+    margin: 18px 36px 40px;
+    box-shadow: var(--shadow2);
+    overflow: hidden;
+}
+
+.ftk-left-section {
+    display: flex;
+    flex-direction: column;
+    width: 33%;
```

Comment:
Not sure where this is used yet, but flagging it as it seems suspicious to use a 33% width on a major section. It would be helpful if the class name was more explicit.

It may be fine. Just seems odd. If you feel it's okay, then that's fine.